backend/src/posts/dto/create-post.dto.ts

```diff
@@ -0,0 +1,5 @@
+export class CreatePostDto {
```

Comment:
Class schaut gut aus. Allerdings werden die Typen nicht automatisch geprüft. Es wäre wichtig hier den Class-Validator zu verwenden. Eine POST request, in der z.B. der title fehlt würde nur bei Verwendung des Class-Validators abgelehnt werden. 

Siehe Bsp hier: https://github.com/AMOS-ConversationalTax/ConversationalTax/blob/develop/backend/src/lang/lang.dto.ts

Doku: https://github.com/typestack/class-validator